website/content/en/docs/getting-started/getting-started-with-karpenter/_index.md

```diff
@@ -145,19 +157,63 @@ The new stack has only one user, `admin`, and the password is stored in a secret
+
+```text
+com.amazonaws.<region>.ec2
+com.amazonaws.<region>.ecr.api
+com.amazonaws.<region>.ecr.dkr
+com.amazonaws.<region>.s3 – For pulling container images
+com.amazonaws.<region>.sts – For IAM roles for service accounts
+com.amazonaws.<region>.ssm - For resolving default AMIs
+com.amazonaws.<region>.sqs - If using the [interruption handling]({{< ref "../../concepts/disruption#interruption" >}}), to access interruption messages
+```
+
+If you do not currently have these endpoints surfaced in your VPC, you can add the endpoints by running the following command.
+
+```bash
+aws ec2 create-vpc-endpoint --vpc-id ${VPC_ID} --service-name ${SERVICE_NAME} --vpc-endpoint-type Interface --subnet-ids ${SUBNET_IDS} --security-group-ids ${SECURITY_GROUP_IDS}
+```
+
+{{% alert title="Note" color="primary" %}}
+
+Karpenter (controller and webhook deployment) container images must be in or copied to Amazon ECR private or to a another private registry accessible from inside the VPC. If these are not available from within the VPC, or from networks peered with the VPC, you will get Image pull errors when Kubernetes tries to pull these images from ECR public.
+
+{{% /alert %}}
+
+{{% alert title="Note" color="primary" %}}
+
+There is currently no VPC private endpoint for the [Price List Query API](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/using-price-list-query-api.html). As a result, pricing data can go stale over time. By default, Karpenter ships a static price list that is updated when each binary is released.
+
+Failed requests for pricing data will result in the following error messages
+
+```bash
+ERROR   controller.aws.pricing  updating on-demand pricing, RequestError: send request failed
+caused by: Post "https://api.pricing.us-east-1.amazonaws.com/": dial tcp 52.94.231.236:443: i/o timeout; RequestError: send request failed
+caused by: Post "https://api.pricing.us-east-1.amazonaws.com/": dial tcp 52.94.231.236:443: i/o timeout, using existing pricing data from 2022-08-17T00:19:52Z  {"commit": "4b5f953"}
+```
+
+{{% /alert %}}
+
+### Preventing APIServer Request Throttling
+
+Kubernetes uses [FlowSchemas](https://kubernetes.io/docs/concepts/cluster-administration/flow-control/#flowschema) and [PriorityLevelConfigurations](https://kubernetes.io/docs/concepts/cluster-administration/flow-control/#prioritylevelconfiguration) to map calls to the API server into buckets which determine each user agent's throttling limits.
+
+By default, Karpenter is placed in the `workload-low` PriorityLevelConfiguration for all APIServer requests. This means that other components that make a high number of requests to the APIServer may affect the ability for Karpenter to make requests.
```

Comment:
is this only when Karpenter is not deployed in `kube-system`?

Reply:
Oh looks like only leaderelection is prioritized when in kube-system

Reply:
Both are prioritized. If you look at the FlowSchemas, you can see that there are mappings to leader-election for leader election requests and workload-high for all other requests with the `workload-leader-election` and `kube-system-service-accounts` FlowSchemas

Reply:
Yeah, I meant if you don't have a FlowSchema at all and just put Karpenter in kube-system, leader election is prioritized but not other traffic.